### PR TITLE
Change link requirement into strats: Inner Green Maridia

### PR DESCRIPTION
**Pull request by fdtaillefer**

### CODE REVIEW

File: region/maridia/inner-green.json [hunk 1]
```diff
@@ -132,57 +166,131 @@
...
+                        "fromNode": 2,
+                        "framesRemaining": 30,
+                        "shinesparkFrames": 23
+                      }}
                     ]}
-                  ]},
-                  {"and": [
+                  ],
+                  "note": [
+                    "Not sure whether this should require canNavigateUnderwater?",
```
Comment (osse101) [inline region/maridia/inner-green.json]: It does make use of water physics to reach the top platform 🤷‍♀

File: region/maridia/inner-green.json [hunk 2]
```diff
@@ -878,88 +924,236 @@
...
+                {
+                  "name": "Suitless",
+                  "notable": false,
+                  "requires": ["canSuitlessMaridia"]
+                }
+              ]
+            },
+            {
+              "id": 7,
+              "strats": [
```
Comment (osse101) [inline region/maridia/inner-green.json]: missing gravity+springball

File: region/maridia/inner-green.json (hunk 1 above)
Comment (fdtaillefer) [reply region/maridia/inner-green.json]: Between that and the fact that positioning the spark can be finicky depending on incoming speed, I'll just require it.

File: region/maridia/inner-green.json (hunk 2 above)
Comment (fdtaillefer) [reply region/maridia/inner-green.json]: Sure is

**Pull request merged**